Adapter/SymfonyCache.php

```diff
@@ -65,16 +70,27 @@ class SymfonyCache implements CacheAdapterInterface
      * @param bool            $phpCodeCacheEnabled If true, will clear APC or PHP OPcache code cache
      * @param array           $types               A cache types array
      * @param array           $servers             An array of servers
+     * @param array           $timeouts            An array of timeout options
      */
-    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers)
+    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers, array $timeouts = array())
     {
+        if (!$timeouts) {
+            @trigger_error('The argument "timeouts" is missing, please inject it.', E_USER_DEPRECATED);
```

Comment:
"The "timeouts" argument is available since 3.x and will become mandatory in 4.0, please provide it"